tests/nightly/JenkinsfileForBinaries

```diff
@@ -55,40 +55,40 @@ core_logic: {
       node(NODE_LINUX_GPU) {
         ws('workspace/nt-ImageClassificationTest') {
           utils.unpack_and_init('gpu', mx_lib)
-          utils.docker_run('ubuntu_nightly_gpu', 'nightly_test_image_classification', true)
```

Comment:
It's better to use `ubuntu_gpu_cu101` instead of `ubuntu_build_cuda` throughout.

Reply:
I tested ubuntu_gpu_cu101 first, most cases failed due to lack of some runtime libraries.